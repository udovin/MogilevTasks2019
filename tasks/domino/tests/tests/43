890
-4011954 12203
-13081324 45388
11880980 35848
-9677269 3773
4091102 19356
-13134224 5538
14454744 13087
3354380 921
-7043581 20451
-13376420 36090
10814463 6002
10490802 30014
6571867 36052
14528777 25413
18765747 3913
-3303535 13435
18090819 13550
5991576 12347
9283739 7628
-8249799 42426
-3798565 3161
13887884 25776
2039435 3192
5425644 2116
10352869 54857
-3685728 19138
-13867134 38403
-2788161 30263644
-7485467 21634
5767119 35297
-11382813 25558
13858312 3256
5125872 18154
9787725 20580
-1422413 15973
2794337 8864
-4253135 4651
706461 5133
15848480 472
7778538 7596
-6606728 932
12499346 12071
3208226 16500
-10337363 12515
8354503 40029
4457259 5486
12016329 8042
825462 5359
5402101 21737
1826575 1969
648751 14292
14863943 38511
8541593 1964
-4792260 5412
-6145746 11450
8670325 30310
-8190477 2201
-4418989 64034
-8424720 11380
-6392425 8072
356029 17528
5353235 10784
9739434 19410
-4228544 6375
1514187 8226
-7272699 31919
-11088201 10274
16172222 24366
-1813774 21080
-3000229 36404
13178591 1066
-7174197 1744
16877325 4964
7333426 39442
-3091639 2693
-9237163 6662
107520 2065
-9336414 2646
-2398399 14398
15947353 54239
-5088895 3797
-10828382 40213474
12753686 5002
-8536598 45944
-611326 15195
-4577728 38631
-9431854 42043
-2325164 23611
15363169 31006
8535388 15457
-10820658 6330
-6733721 42437
-6105889 16067
-2666804 39010
-10585463 7682
14902230 33701
17203414 20448
18261643 4493
11979963 4831
456021 6988
17039758 13844
8040479 10707
929800 17601
9524989 33992
10058149 6606
-13486893 11380
-2478704 50589
2278065 7662
-527304 37794
-6908913 13791
10751544 49214
-11649100 1358
-10553249 17027
-5776522 14
14050061 26898
-5432754 5468
13033082 17909820
10591796 99764705
6898743 1225
14487185 4649
-10595354 15782
3257567 7467
-10500022 5247
-8597005 15590
-6848425 29811
6374753 38559
11288678 15387
10175633 534
18510081 3598
5593201 10356
-7891599 3504
-9405831 18760
6599598 289
-4024861 31427
14268998 4793
3089846 23643
-6051577 3737
14101064 15815
17967045 10701
-93744 39427
-6404624 24566
-12290752 28806
-10760828 11609
8309717 5366
2320050 13227
1346916 55423
2585191 5398
-3869425 1957
9222314 2369
-7469540 8716
-6899762 4085
-8795469 5540
10649250 15853
672626 7786
1876177 1190
-985227 9272
15150744 24975
4760955 15168
1499158 11568
-4389601 5325
-7117788 3479
-2866111 29776
-2231399 37545223
9840644 1174
4820405 12947
9973048 850
4000372 1030
17125902 14396
-10669117 6123
17732127 16217
-11175159 44378
-864849 18561
12858257 16766
1275556 44805
11832773 9012
-92449 553
4515038 38756
-9526157 317
-12192773 7689
9473188 21746
9265751 30637
-11579256 1675094
13660398 52527
-5287156 8398
-3576305 17136
9633599 35743
-5912751 28447
-12718788 39428
12391791 1852
17794862 8105
16598381 5345
-8334523 76686962
11973797 39388
-1174878 27468
13732949 37338
10956517 1381
2091882 7124
-6682232 6797
-13821110 20221
-13269080 25442
-677204 36253
-10903105 3403
-4075323 37209
18654439 3366
17421025 5676
3588582 18158
3701381 21823
10121615 32590
10942686 4315
-48808 4379
-9102587 3294
16414484 27657
255362 17135
-10889808 5431
8424697 565
-9323706 11712
12149824 34397
-5018130 37894
14141650 20951
-13329264 34853794
8096026 10993
16804713 4509
-13917661 1194
-2720630 9467
7073698 32476
-4866111 28885
-2367880 3293
13551371 6698
-4123709 1719
6046657 11857
-6501738 25531
12474102 7125
584712 11696
16946887 19002
-2735786 26171
16640737 3987
-7319686 41564
18059757 36030
1984667 27466
539826 1410
13747447 10970
13214050 2367
6707467 32239
-10380544 8397
17619450 3288
-54002 34659
-1292163 6928
8300139 778
2731494 19319
6030169 13638
1051681 87142312
-11986026 49426
-2770310 1759
9560512 319
16213562 35507
11993826 9955
-8309482 5734
12485274 5167
14198981 3338
-10423184 7980
3765100 17803
-1252766 3868
-7794580 13418
-3641752 27682
8223849 18966
10297980 23945
-11651841 17364
16859352 20256
15419381 10610
-12100971 17229
-6575933 1105
1567629 44333
4070185 3553
10252224 1248
-10464209 11271
4618374 13675
-10697432 30410
13521604 32484
16435372 1243
4292713 23199
-4927164 26960
12152896 3053
9091950 19738
11583756 17618
-6241941 22121
18537782 1634
18750777 37
-4661693 16562
-1937379 17758
-11890379 1870
1471772 55735
3788352 9964
18556090 5600
14331799 61090
-5485419 438
-1046208 12624
2947205 4953
10250487 20215
7853599 40516
-11411367 18562
11100590 3471
12350255 52996
11382271 15111
7937778 67335777
-13793521 18560
6949599 14446
-4560578 15078
-7998336 28806840
15695083 538
-5751204 14100
-5971228 2690
10695513 2157
-8935207 694
16013121 17074
15401764 12433
6440670 4220
8933392 1023
-13647543 47774
16378826 9347
-4068734 983
11631855 30913
-1765908 6001
12224513 53238
5247393 4041
-5178383 3545
3271256 7654
-9831058 20510
13450928 6255
6591484 12649
6835659 4800
11348330 22394
-9045249 35285
-10003204 2061
12437840 12016
6206665 5789
-1722051 25351
-5200391 22374
-13046856 33006
13566475 4431
-2059009 4974
-2106965 36233
15530772 9833
19276461 1992
10051400 17809
-588618 1417
10172723 965
8171323 142
-5777154 16725
-1080689 5400
-3809394 57503
2241043 24059
-9498021 13935
1860701 3317
10602039 9637
11006448 39379
-3115762 5823
-9161412 30010
4562555 2076
18335375 5336
4162855 41097
-13557777 1638
14687824 41889084
17894996 30420
12474298 112
948002 9855
12379461 675
9203788 18762
15248531 18137
322336 50345
-7178311 26186
-7217473 3899
2343031 5835
-7734754 30838
13434371 3889
-3942047 28245
-8803555 45730
11068808 50907
17414872 18410
12388463 1092
11397111 787
-13580108 11110
-10006671 7798
17465539 17087
-4799512 4667
19129382 27076
-2181061 4885
-5445200 10014
6326194 8862
-5963393 4339
6272861 11464
17797501 2339
874255 33946
12170513 17380
-8477704 18773
-239176 38024
-10255500 25632
1219996 13094
-11905568 60687
-3477052 66868090
12256606 9657
17105861 22291
-11762571 50082
-13612297 7988
12040418 8949
-10878151 99
14396458 46045
14700637 6423
13077404 653
-7896654 18475
3473072 39901
187117 580
18937100 30916
16712505 5424
-1638642 63908
12921246 362
-4985087 3562
-10915076 15978
-13419294 56691
16077153 15409
9000473 7091
4234031 3373
-11431986 48478
-5373741 7421
-7161491 1826
16232619 4036
14758079 23016
5237826 5666
1939840 38400
-6009867 31279
5570166 2645
15582250 40254
18154787 64420430
-4312902 2935
-7924470 64907
-6179723 8310
7657673 61849
15256016 183
-7864461 21681
-6779159 33395
2517257 30882
-4331521 5773
-6133502 6806
2520437 1416
-8053307 15865
-12993630 40133
7014142 30620
14761009 2840
15783227 15390
5898915 15080
-11004375 26269
2659326 37768
5298737 22828
14263656 54926
-6297008 23235
-11824470 16375
14812159 6432
17329936 9398
3427168 58930
-3424733 19153
-10014843 22222
-12330281 8490
12603206 32663
-12364380 21258
11463572 51720
-13841513 1576
17133071 5940
5831843 4564
2861182 6176
-11323642 28890
-11228979 9339
12091556 62299225
15665222 10224
-11722152 23660
1804682 8007
-10110547 7646
-5535893 3923
6473372 14018
-2202727 22045
-11109671 5388
12912879 9934
3146675 41467
8754126 37420
11763049 46622
9838858 15616
12020040 3372
-2628318 12209
-3749311 28464
11182608 5013
-7163963 7466
3778351 10716
-382121 58960
996086 19530
-9201460 487
-4725032 79241537
10912145 187
9947461 43205
-3539352 8020
3012163 999
-2541500 18845
76458 17920
9030788 1291
12743082 556
-8049029 3981
472412 8912
511299 689
2937358 27323
11690611 21746
-6973601 41757
-1784193 9441
-4072164 2216
13074621 36872
2812145 11576
537189 16521
15606362 8849
-7426785 29370
13287393 40459
5029490 50191
15689423 639
-9772791 788
9167678 3179
5954456 5966
-2289385 23365
7286197 9155
7525998 15177
11225760 24041
6117640 64670428
10147945 25356
-7662197 62384
17378602 20386
7782034 1023
5609004 14452
5677252 19412812
-1317378 30260
14730300 9966
-8126675 46487
4536790 6427
18448068 169
3353005 13911
423842 19813129
-12949453 36576
2373281 20507
-9372863 19968
11388384 5038
14564586 497
-1890528 10841
19197424 14998
-2229770 128
1651764 19333
692205 10316
-10193468 15315
7126337 99046988
7874004 14231
810232 71735
3653732 48300
3519790 33474
-13414341 4800
15892421 11271
-4618735 22669
-1310916 2680
13214191 39
-4191761 2811
16355322 49871
11807977 33166
19073707 16248
14647415 42398
-5133830 22086
17013265 41730
-12201345 27010
-13210879 47074
-12554418 27868
4732518 56075066
16947208 185
-5621606 33323
5488407 58771
15493259 58578
18295213 32159
-10120543 42036
-6063587 40366
10023944 24837
10548934 36420
2391654 1054
4671785 45773
563566 3040
-11495421 13225
12362568 5922
4541381 2092
2767519 18631
-1748026 11031
13801210 37672
-9972591 7833
-11746005 3787
4307304 1501
3021838 3800
-8945985 30840
-4324118 5449
5820582 16730
-1765518 298
-5841979 68806
-6578847 14551
19098195 15518
19232912 16654
10888963 19729
-448523 5018
3669600 6240
-1857087 8792
11835385 772
-4298943 9329
-12657102 39654
15788442 4228
18436494 32024
-743205 264
-501737 15756
13050 4037
14149809 7785
-12156782 30627
-745296 7824
18446150 2760
-841107 13964
15841933 24382
560268 1748
8490355 38568
15989725 11866
-9542692 59829
9875754 20310
-593646 2553
-1484032 13153
7590072 2001
-9933416 16729
15795697 4772
-4258716 5678
12802177 18301
1758925 37705
14601456 7360
-8316494 1983
-518388 4278
3860831 5249
2835111 17252
-299965 34703
8547481 1081
8298341 34024
14332101 150
148520 11175
8606284 2195
18649838 39704
11924488 25364
11469899 3393
-6641419 26285
6771307 11069
7200455 58174
-136891 52436
-1498343 141
-2129 8524
-11845947 19245
9479441 3332
6183193 50963
-2414043 13561
13606427 15000
738410 19126
-12775614 13156
16098995 846
12265879 5383
-3341039 50
-8873031 60641
591478 2918
-789108 9782
19204142 5970
3302698 23957
19266277 29727
5704882 17596
14591040 21778
5879150 39535
17307794 10422
-6469579 8651
502128 16463
1602047 31602
-479992 18001
17946732 45198
-6328860 11345
-5333986 18526
-5685602 13442
-7645105 4819
-13718941 71445
10416321 34770
7397392 55462
-2372689 20545
16777256 14792
-10968433 23730
11531000 4100
13361642 642
-8197743 51150
12669252 29147
14115338 10588
4212652 43824
6012224 2778
-2195683 3906
6526789 25234
14548525 3209
-5392346 14533
2903357 74650929
17849827 41284
-4804221 38625
6300131 25341
5173131 19547084
7270328 4370
5321146 1226
-13204503 682
7500779 24862508
-6923500 37189
6407012 13276
-1712211 4572
2468192 44338
6197235 9563
14710765 7848
-3908799 19864
15110458 57917
18349218 2446
8082016 21571
-6353764 5733
-338770 21428
-3258091 29697
9692263 34869
-3188880 28359
1188040 49341
4763971 2645
-1996344 9550
15311802 5793
16559337 22165
11632296 375
14006752 5942
-3548609 23035
-13911531 16
-3352167 46465
18022607 11706
19001198 62913
-2926564 471
11169040 55066
485326 4811
-1853678 1885
8793354 29725
-2858575 5572
-10797008 5779
1712165 50700
17241228 13032
12277668 1276
-11889026 156
19211562 4083
-11250163 72033480
18805141 12799
-2560547 49780
810588 250
12963128 10542
17259004 5601
9559626 14449
7444440 1304
-12401286 9919
-10324719 4683
-7104130 6050
-8586219 3850
16041464 3809
16287063 52198
-10807368 6877
18578576 10778
4576723 2143
16562483 111
-201217 705
-12883461 445845
1121398 29473
-11529577 5707
-12246149 29886
7262171 7809
5318527 7392
18872010 28208
6904950 2432
9163140 36473
18379175 25759
-12833909 37064
2080599 40848
5378420 11219
-9605048 19833903
6667666 46954
18750544 36470
12742057 7477
10910627 12890
-896832 25051
12544506 34369
-1613728 192
2579425 648
8415588 20240
-2662903 947
-4486259 70465
9002882 2082
-12518767 17913
3885402 7106
-5662423 15478
13133361 39734
-12604020 14538
9395774 29010
6824435 51755
-10038936 24866
4957392 64272
-215297 7632
-1886111 2176
-11032964 61007747
8114691 7070
17170986 5868
-5795922 42128
-8405162 11297
9575125 1516
616002 53
104916 10757
-5259803 18316
-942022 7955
-1539861 18532
-12069024 22591
4892597 37206
-8981960 53403
10804362 46679
2394230 2492
13945222 36134
4428952 54353630
8044567 2711
18708439 25504
14973663 28261
9347307 63237
16684011 39861
-9264746 813
17496203 14247
-12040328 23373
-805419 19890
-3977487 12516
-7606347 14070
3559454 20955
-9747232 6636
9028713 11724
2135861 7372
17658243 4531
-10794364 2444
-7557279 6656
-1503777 20629
9401647 2
13155581 10954
-5115307 3849
18217777 23901
-13172509 30526
-3046073 42864
4360577 8871
16783561 3206
15216238 53272
7979166 3094
8865561 25675
7242047 774
15728150 18647
2190271 22355
-7380462 23614
-11362094 13119
-8666632 38943
-12240450 2335
17550698 30222
8167871 21496
5100213 14637
-12451025 7399
-1349038 40264
131965 9794
-8734821 38940713
-1167912 88
16921479 4033
3741909 22010
-3122918 36804
-11541555 28208
-644241 24940
15654360 33794
5231938 21905
-11972186 181
16505433 37101
-713927 25435
-13140805 24844
8709747 29444
-5560574 55331
6430317 736
3933484 40719
-1217494 17341
-1098552 10911
5551952 17573
1373861 12213
15044231 31643
1399293 12469
-9864160 22071
10937367 6863
7717885 51369761
13854025 93868318
