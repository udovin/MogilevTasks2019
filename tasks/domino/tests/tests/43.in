441
-58595005 12837
-78921409 68277
-59774739 57704152
-64646687 86253
-62743211 1524
-79906831 125650
-63181443 94874587
-67889024 69937
-62872459 24
-60587717 81686
-64276621 58479
-55746666 43844
-81499277 2332
-68264829 5277
-62278814 37723
-70179868 19244
-70396967 14295
-63602805 97317
-51739230 65976
-79390502 78636
-74655751 77465
-84178464 48041
-80974283 45388
-81594065 92462
-71441247 8678
-56496229 3943
-76229437 73650
-60731365 2699
-78367906 51682
-84420813 40703
-67216763 1697
-79657876 41072
-67225312 10929
-65889725 127907
-58521956 38604
-52292546 8690
-65267572 6115
-84157013 4282
-82049315 28751
-74897033 69545
-80834239 2939
-57928585 69857
-64824192 66366
-77176134 27426
-82570951 15927
-70236175 2828
-73921241 16275
-79240694 19384440
-84112435 10570
-55699943 7048
-54903140 59102
-71821557 61007934
-59037852 39568
-53083796 8003
-65748002 50435
-55383116 372
-68859953 3796
-54574874 37090980
-53720983 3852
-69703640 85085661
-73073357 6004
-52380918 85792
-62346679 1416
-54435436 5271
-70176866 524
-55158475 65893
-56104352 88512
-51819162 71024
-79653299 2820
-69367348 2262
-61471327 62945
-64752865 26164
-58047713 8765
-67520160 8105
-54570090 1042
-76600811 11933
-69159009 49768
-68549615 134
-55686067 10725
-70744696 82479
-81159814 5740
-61544066 31141
-64899261 61189746
-55731379 8500
-59591285 12000
-83815175 26821
-54164681 127720
-61750308 85692
-79748552 5029
-66846728 755
-53837986 41521
-71291230 121487
-76770987 8367
-76816951 6421
-62573376 14409
-68135641 27999
-64552310 29298
-57147200 17325
-72296340 3817
-56471727 9551
-67164922 23295
-53472123 78968
-84404944 15570
-66367550 55726
-61041624 5968
-54056728 19402
-69273460 71897
-75168176 71181
-76856154 58403119
-65275870 19387
-53127377 20716
-52248422 3142
-72791830 20730
-82106115 21784
-82312930 55995
-62755150 77658
-70834581 59273
-52145056 9191
-55795183 8692
-63282874 28075
-84246177 20859
-71987415 127813
-79129616 106153
-75667876 65703
-66471474 60701
-83981263 8251
-59704866 61769
-84348708 28503
-55378100 1052
-76134608 7571
-66931005 98341
-68901356 17657
-68876363 7746
-78756626 40818
-70687550 21953
-52013472 39380
-52348896 19725
-77606285 17903
-79755050 32279
-72255637 11344
-78293653 5234
-83945831 17616
-77229030 110520
-55009866 7236
-72305386 51860
-81505605 10986
-65471516 20168
-82373635 13665
-61049067 11250
-80558317 29610
-80623731 74541
-73544077 113895
-64066084 50349
-59548474 782
-64172642 15550
-70983167 15990
-81723938 1410
-75907649 25242
-71906599 26743
-61589234 129165
-62482015 14455
-64433033 6795
-81533768 57003
-60706441 2224
-66024349 30130
-73491789 46291
-77175751 348
-67319536 6248606
-58699714 47363
-68992101 44501
-81740489 57814
-84313543 17109
-59407132 23189
-54557266 1640
-80172954 11156
-59355756 16977
-81811107 39269
-62626536 25161
-76155927 42245
-55541610 27274
-55280837 4318
-74762514 44613
-60468160 22373
-69069210 42565
-67030692 55023
-70095648 15017
-56759663 112931
-73325655 80585
-60037256 67752
-57110813 25534
-62143252 24780
-54829244 14271
-69383443 125875
-80086058 67634
-53983835 6071
-72595183 38582
-69635323 56251
-52965476 110951
-66659801 15309
-80260819 23248
-76651423 37549
-52858707 43711
-83408576 59526
-82164637 55844
-57312116 74730
-52618333 782
-75533513 19565
-79477970 33578
-82709857 52
-71514465 58873
-78325120 18040
-73155358 35977
-55937166 52011
-82081930 10581
-68755109 85349
-62742877 320
-75413904 69417
-78586016 22699809
-56740316 8341
-64780769 9868
-78863253 40434
-62206157 11683
-59289408 53332
-68034554 28534
-62935132 29245
-55869515 67562
-56248742 63720
-79031748 97841
-78114490 254
-52054000 70162
-65404666 58586
-63048145 19443
-82949546 35058
-59549746 6738
-60161230 115609
-52501671 58600
-57172304 9092
-63436632 123360
-72573473 929
-63566662 34990
-68335953 11432
-71607176 135295
-77744212 123758
-67854378 2060
-75558242 59428
-70219395 16281
-82791106 5563
-64734004 10822
-64535527 12234
-70583372 9048
-78132355 66534
-83032626 13476
-74546324 4589
-78252917 2118
-58137481 41142
-59166865 5446
-56592714 143999
-59556974 25168
-65164202 41563
-83905965 35314
-63981015 54074
-79795309 103831
-76339938 13030
-77637328 90110
-77077031 56729
-68409150 81596
-63566078 227
-78480694 39346
-67677079 25430
-68671057 4617
-82807571 47547
-76999725 14777
-83277093 20266919
-63849734 12634
-65215357 32686
-76692945 33602
-80056215 28924
-75104017 26967
-74047872 47709
-56533274 10302
-62026473 114042
-60842468 10385
-81372099 2591
-54306656 97871
-57418601 103739
-67239592 33957
-55397410 102051
-83404905 779
-56348731 68517
-80840244 120896
-57020171 72545
-74539228 4011
-81192421 9786
-58382124 137242
-66762791 41409
-73817033 63191517
-74238785 1305
-57781667 26135
-83636875 63665
-57808241 60485
-75790696 32169
-70608211 65449
-72959400 71303
-67530303 82543
-71147396 26810
-74129722 20119
-78534062 20393
-60854253 34651
-73106759 47773
-60918664 110175
-72254293 117
-69509327 55703
-80360897 1697
-54847671 44873
-54686223 70537
-76364169 114717
-61220669 14314
-60305947 2880
-55383952 1347
-56048372 2145
-61112186 30443
-72839139 41859
-51890244 101357
-72137620 2593402
-58189551 93042
-83488210 4101
-53608631 57438
-75830582 63956
-72288375 6450
-63733033 13370
-80367763 92269
-80471067 61447
-58638747 58142
-73672169 41110
-71459856 51221
-72658164 66302
-65619660 4840
-75025275 44246443
-75274212 3690
-81310979 59189
-67424447 11617
-63409173 13376
-58901158 30685
-68165711 37773
-69928755 102790
-78021469 13406
-74250223 102431
-71054953 81782
-59247590 15198
-79550828 35770
-83125578 72353
-58294812 56001
-52707974 55993484
-73947457 55392
-81731417 6556
-58772194 37411
-69364821 2366
-61732287 4571
-83747011 18005
-66158642 5106028
-76496575 27396
-69622203 5837
-81107306 21628
-76054609 790
-70077352 4952
-82440955 80673
-74038019 5722
-60027304 7919
-56906076 59366
-59910688 18980
-58853263 34837
-73203973 21140
-61894367 125682
-61297349 31773
-81205732 21212601
-61411765 42590
-53768581 22973
-75779506 336
-77514714 44758
-66271878 52744948
-59523812 24255
-73407821 81097
-78037333 44884
-68689136 59892
-65368092 26356
-70555323 9477
-70415841 87484
-78676932 31083
-61092402 2896
-60743825 86902
-63971863 1754
-83232897 14823
-60334998 28120
-69820056 7327
-53327641 11048
-55075860 79985
-65697659 1064
-66441505 1346
-67157948 4723
-54442121 68277
-56442439 28708
-65851925 14645
-78708699 12093
-67820365 13046
-82861958 66548
-66863082 27431
-52649155 16524
-65469887 266
-80766311 61156
-74558033 13387
-71752479 38309
-77367374 60054
-80705181 60543
-57674508 6429
-83493932 50515
-74398677 112336
-81381481 29848
-74187258 16958
-62913399 20822
-57568153 55236
-78470291 757
-72451609 43528761
-77873146 129955
-82403709 6972
-70258653 67308
-70978469 410
-75138430 28732
-68549829 44986
-64417211 112
-82728371 47101
-71014144 14301
-78473842 5291
-53223713 38780
-62668648 54766
-66576392 15930
-63794078 16015
-81931534 38833
-65015572 12626
-65136064 21339
-70199630 17494
-52185110 58853
