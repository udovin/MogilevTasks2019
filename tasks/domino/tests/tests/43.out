1 2 103 1 2 4 147 3 1 1 1 2 1 1 1 2 1 3 1 1 1 2 1 4 1 1 3 1 2 2 1 2 2 2 1 1 1 1 1 1 1 1 2 2 1 1 1 365 1 1 3 267 1 1 2 1 1 35 1 237 1 2 1 1 1 1 2 1 1 1 2 2 1 1 1 1 1 1 1 2 1 1 171 1 1 1 2 2 1 1 1 1 1 1 1 1 2 1 1 1 2 1 1 1 1 1 1 3 333 2 1 1 1 1 1 3 1 1 1 1 1 2 2 1 2 1 1 1 1 1 3 1 1 1 1 1 1 1 2 2 1 1 3 1 5 2 2 1 2 1 2 2 1 1 1 2 1 1 1 4 1 1 3 1 1 1 1 205 1 1 3 1 1 1 1 1 1 1 2 1 1 1 1 1 1 1 2 1 2 1 1 1 4 2 1 2 2 2 1 1 1 1 2 1 1 1 1 1 1 3 1 1 1 1 3 1 1 357 1 1 1 1 2 1 2 1 1 1 1 2 2 1 1 2 1 1 1 2 1 2 1 4 3 1 2 1 1 1 1 1 2 1 1 1 1 1 4 3 2 1 2 3 1 2 1 2 1 3 1 1 2 1 423 1 1 1 1 1 2 1 1 1 1 1 1 3 4 1 1 4 1 1 1 2 1 292 1 1 1 2 2 3 1 2 1 1 1 2 2 3 1 1 1 2 1 2 1 1 2 1 2 1 2 270 2 1 1 3 1 1 2 1 2 1 2 3 1 308 1 1 1 1 1 2 1 1 3 4 1 1 1 1 15 2 1 1 1 1 1 188 1 1 1 1 1 3 1 1 1 1 1 1 1 1 392 1 1 1 1 189 1 1 2 2 1 1 2 1 1 3 1 1 1 1 1 2 1 1 1 2 1 1 1 1 3 2 1 1 2 2 1 1 1 1 2 1 2 1 1 1 1 276 4 1 4 1 1 2 1 2 1 2 1 2 1 1 1 1 1 1 2 
